androidtv/basetv/basetv.py

```diff
@@ -472,6 +473,29 @@ def _running_apps(running_apps_response):
 
         return None
 
+    @staticmethod
+    def _installed_apps(installed_apps_response):
+        """Get the installed apps output of :py:const:`androidtv.constants.CMD_INSTALLED_APPS`.
+
+        Parameters
+        ----------
+        installed_apps_response : str, None
+            The output of :py:const:`androidtv.constants.CMD_INSTALLED_APPS`
+
+        Returns
+        -------
+        list, None
+            A list of the installed apps, or ``None`` if it could not be determined
+
+        """
+
+        if installed_apps_response is not None:
+            if isinstance(installed_apps_response, list):
+                return [line.strip().rsplit("package:", 1)[-1] for line in installed_apps_response if line.strip()]
```

Comment:
You should remove lines 493 and 494, the output of a shell command will always be a string (or `None` if it fails), it will never be a list.

Reply:
Sure! I was copying what was done for running apps, didn't fully get why it was done that way over there.

Reply:
Adjusted! Will rebase and squash everything as a final step just to clean up the commits. Thanks for all your reviewing.

Reply:
You don't need to squash, I'll hit "squash & merge."

Reply:
> I was copying what was done for running apps, didn't fully get why it was done that way over there.

I see. Yeah, that's not necessary, I'll fix that at some point.

Reply:
Perfect! Thank you.

Reply:
FYI, the running apps function actually does need to be able to work with a string (in the case that the `running_apps()` method is called) or a list (when used as part of `get_properties()`).  This is because when parsing the properties, it forms a list by splitting the lines of the running apps output.